src/Sema.zig

```diff
@@ -6530,7 +6530,7 @@ fn zirDeclRef(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!Air
         .no_embedded_nulls,
     );
     const nav_index = try sema.lookupIdentifier(block, decl_name);
-    return sema.analyzeNavRef(block, src, nav_index);
+    return sema.analyzeNavRefInner(block, src, nav_index, true);
```

Comment:
nop diff, suggest reverting

Reply:
Intentionally kept to prefer shallower call stacks

Reply:
I think it just comes down to taste, and if there's a disagreement then semantic compression should be the deciding factor, so I'll concede this one